### ISSUE TITLE
Indent issue in "with" having multiple clauses

### ISSUE THREAD

**Issue by gmile** [elixir-editors/vim-elixir#339]
This is what I'm seeing with the latest `vim-elixir`:

# Actual

```ex
def my_function do
  with :ok <- some_call,
       :ok <- another_call do

       end
end
```

# Expected

I expected this intuitively, but [lexmag/elixir-style-guide](https://github.com/lexmag/elixir-style-guide#with-indentation) also mentions the same style:

```ex
def my_function do
  with :ok <- some_call,
       :ok <- another_call do

  end
end
```

**Comment by drewolson** [comment]
I'm seeing the same issue.

I'm seeing this:

```elixir
with {:ok, foo} <- thing(1),
     {:ok, bar} <- thing(2) do
       foo + bar
     end
```

I expected:

```elixir
with {:ok, foo} <- thing(1),
     {:ok, bar} <- thing(2) do
  foo + bar
end
```

**Comment by jbodah** [comment]
Thanks for these - these are definitely bugs. `with` blocks are tricky but I'll see what we can do

**Comment by jbodah** [comment]
Fixed in 1543e37f5940396b3cd18f67f1c36530d289314c
